src/Nancy/Request.cs

```diff
@@ -316,5 +318,13 @@ private void RewriteMethod()
 
             this.Method = providedOverride.Single().Item2;
         }
+
+        private string DebuggerDisplay
+        {
+            get
+            {
+                return String.Format("{0} {1}", this.Method, this.Url);

```

Comment:
One question, what do you guys think it's better:
{0} {1} or Method={0} Url={1}?
I would use the same format on the RouteDescription DebuggerDisplay